stories/collective-page/Hero.stories.mdx

```diff
@@ -0,0 +1,27 @@
+import { ArgsTable, Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+import Hero from '../../components/collective-page/hero/Hero';
+import { webpackCollective, openSourceHost } from '../mocks/collectives';
+
+<Meta
+  title="Collective Page/Hero"
+  component={Hero}
+  argTypes={{
+    collective: { defaultValue: webpackCollective },
+    host: { defaultValue: openSourceHost },
+  }}
+  parameters={{
+    actions: {
+      handles: ['mouseover', 'click'],
+    },
+  }}
```

Comment:
Unless I'm missing something, I don't think logging these events is really interesting, is it?

```suggestion
```

Reply:
Yes we could remove these. I've removed all of those. 😺